entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# set web server port
+web_server_port=${1:-8080}
+
+# update configuration files and initialise DB
+"${SERVICE_PATH}/setup.sh"
+
+# start the web server
+"${SERVICE_PATH}/start.sh" ${web_server_port}
```

Comment:
This whole process seems simpler and with less commands than the longer starting line that we had in the helm chart for portal. Good! Are all these commands available in setup.sh and start.sh or we really never needed all that complication? I wonder...

Reply:
The long list of commands of the old starting line has been reorganized in a more consistent way on the two setup.sh and start.sh scripts: the former contains all the logic to configure the MySQL connection and to initialize the MySQL DB; the latter simply starts the PHP daemon for serving the App.